photon-client/src/components/dashboard/CamerasCard.vue

```diff
@@ -81,10 +81,10 @@ const performanceRecommendation = computed<string>(() => {
     </v-row>
     <v-divider style="border-color: white" />
     <v-row class="stream-viewer-container pa-3">
-      <v-col v-show="value.includes(0)" class="stream-view">
```

Comment:
This should be v-show due to often dynamic rendering. Doing v-if adds more overhead and is incorrect for our use case.

Reply:
Maybe use intersection API within photon-camera-stream itself and hide the stream element if `notInView && backendConnected`

Reply:
In my opinion v-if seems fine given it just cleanly fixes the double load problem.